src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/JsonWebToken.cs

```diff
@@ -70,6 +71,8 @@ internal class JsonWebToken
 
         private readonly JWTPayload payload;
 
+        private bool _sendX5c = false;
```

Comment:
not consistent with the rest of the files. pls. remove _ unless we decide that is out new norm